README.md

```diff
@@ -88,6 +89,29 @@ You can assign attributes for your appliances using three methods:
 
 Once the authentication is prepared and the manifests containing the resources are written, you can use ``` puppet apply <manifest>``` to run your manifests and execute the desired changes to the system.
 
+### Synergy Image Streamer Authentication
+
+The attributes required for authenticating with your HPE Synergy Image Streamer Appliance are:
+
+* `I3S_URL` - The web address for the HPE Image Streamer appliance. For example, https://imagestreamer.example.com
+* `I3S_TOKEN` - Session token used for authentication.
```

Comment:
The token isn't the same token the user uses to access the HPE OneView REST API? If so, it could be said here to avoid misunderstandings or wrong assumptions.